controller/ItemImportSampleDownload.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
```

Comment:
We use only this comment here: https://github.com/oat-sa/extension-tao-itemqti/pull/1751/files#diff-011a17d61ee057fb83fa3032ad1b8108034ecae3b80a18bb9072c9fa38a82b10R3

No need to add script author on comments. We do not do that anymore